components/login/index.jsx

```diff
@@ -52,23 +55,46 @@ const useStyles = makeStyles((theme) => createStyles(styles(theme)));
 
 export const AUTH_OPTIONS = {
   clientName: CLIENT_NAME,
+  clientId: CLIENT_APP_WEBID,
 };
 
-export default function Login() {
```

Comment:
Whilst the component is `Login` it's easily confused with `components/pages/login`, and in the place where it's used, it's called `LoginForm`, so this helps with ambiguity in the code.